sympy/stats/tests/test_continuous_rv.py

```diff
@@ -685,14 +711,14 @@ def test_uniform_P():
     assert P(X < l) == 0 and P(X > l + w) == 0
 
 
-@XFAIL
 def test_uniformsum():
     n = Symbol("n", integer=True)
     _k = Symbol("k")
+    x = Symbol("x")
 
     X = UniformSum('x', n)
-    assert density(X)(x) == (Sum((-1)**_k*(-_k + x)**(n - 1)
-                        *binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1))
+    assert str(density(X)(x)) == 'Sum((-1)**_k*(-_k + x)**(n - 1)\
+*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)'
```

Comment:
Or just:
```diff
-    assert str(density(X)(x)) == 'Sum((-1)**_k*(-_k + x)**(n - 1)\
-*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)'
+    assert str(density(X)(x)) == ("Sum((-1)**_k*(-_k + x)**(n - 1)"
+                            "*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)")
```

Reply:
Similarly for `BetaNoncentral`.

Reply:
This is not addressed. Please do not resolve before updating.

Reply:
Sorry about that. I thought you were giving me an option. Also after the above change, my tests are failing;
```
>>> assert str(dens(z)) == "Sum(z**(_k + a - 1)*(c/2)**_k*(1 - z)**(b - 1)*exp(-c/2)/"
    "(beta(_k + a, b)*factorial(_k)), (_k, 0, oo))"
File "/home/ritesh/Desktop/sympy/sympy/stats/tests/test_continuous_rv.py", line 246, in test_beta_noncentral
    assert str(dens(z)) == "Sum(z**(_k + a - 1)*(c/2)**_k*(1 - z)**(b - 1)*exp(-c/2)/"
AssertionError

>>> assert str(density(X)(x)) == "Sum((-1)**_k*(-_k + x)**(n - 1)"
    "*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)"
File "/home/ritesh/Desktop/sympy/sympy/stats/tests/test_continuous_rv.py", line 725, in test_uniformsum
    assert str(density(X)(x)) == "Sum((-1)**_k*(-_k + x)**(n - 1)"
AssertionError
```

Reply:
You'll need to enclose them in braces like the diff for it to be treated as one string.
```
In [1]: ("Sum((-1)**_k*(-_k + x)**(n - 1)" 
   ...:     "*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)")
```

Reply:
Done!